apps/browser-extension-wallet/src/lib/scripts/background/services/cardanoTokenPrices.ts

```diff
@@ -28,7 +28,7 @@ const tokenPrices$ = new BehaviorSubject<{ tokens: TokenPrices } & Status>({
   status: 'idle'
 });
 
-const emitPrices = () =>
+const realEmitPrices = () =>
```

Comment:
I think with this amount of requests debouncing this function will prevent it from emitting any prices for a long time. I suggest using `import throttle from 'lodash/fp/throttle';` instead. 

```suggestion
const throttleTime = 10 * 1000;
const emitPrices = throttle(throttleTime, () =>
```